docs/miscellaneous.rst

```diff
@@ -309,7 +309,6 @@ The following is the order of precedence for operators, listed in order of evalu
 Global Variables
 ================
 
-- ``block.blockhash(uint blockNumber) returns (bytes32)``: hash of the given block - only works for 256 most recent blocks
```

Comment:
Please keep the old one and mark it as deprecated.

Reply:
Do you have a strong opinion on how they should be sorted? Otherwise I'd probably sort them by variable in alphabetical order.